packages/type_eq/type_eq.0.0.1/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "Type equality proofs for OCaml 4"
+maintainer: [
+  "Zach Baylin <zbaylin@gmail.com>" "Trent Small <sixstring982@gmail.com>"
+]
+authors: ["Zach Baylin <zbaylin@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/skolemlabs/type_eq"
+bug-reports: "https://github.com/skolemlabs/type_eq/issues"
+depends: [
+  "ocaml"
```

Comment:
```suggestion
  "ocaml" {>= "4.08"}
```
due to
```

#=== ERROR while compiling type_eq.0.0.1 ======================================#
# context              2.2.0~alpha4~dev | linux/x86_64 | ocaml-base-compiler.4.07.1 | pinned(https://github.com/skolemlabs/type_eq/releases/download/0.0.1/type_eq-0.0.1.tbz)
# path                 ~/.opam/4.07/.opam-switch/build/type_eq.0.0.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p type_eq -j 255 @install @runtest
# exit-code            1
# env-file             ~/.opam/log/type_eq-7-677780.env
# output-file          ~/.opam/log/type_eq-7-677780.out
### output ###
# (cd _build/default && /home/opam/.opam/4.07/bin/ocamlopt.opt -w -40 -g -I test/.type_eq.eobjs/byte -I test/.type_eq.eobjs/native -I /home/opam/.opam/4.07/lib/alcotest -I /home/opam/.opam/4.07/lib/alcotest/engine -I /home/opam/.opam/4.07/lib/alcotest/stdlib_ext -I /home/opam/.opam/4.07/lib/astring -I /home/opam/.opam/4.07/lib/cmdliner -I /home/opam/.opam/4.07/lib/fmt -I /home/opam/.opam/4.07/lib/re -I /home/opam/.opam/4.07/lib/seq -I /home/opam/.opam/4.07/lib/stdlib-shims -I /home/opam/.opam/4.07/lib/uutf -I lib/.type_eq.objs/byte -I lib/.type_eq.objs/native -intf-suffix .ml -no-alias-deps -o test/.type_eq.eobjs/native/dune__exe__Type_eq.cmx -c -impl test/type_eq.ml)
# File "test/type_eq.ml", line 89, characters 34-35:
# Error: Syntax error: operator expected.
```

Reply:
In fact this is only an issue in the tests, so let's not touch this bound

Reply:
Could mark your constraint with a `with-test`

Reply:
It is already done by disabling the tests on ocaml > 4.08, where the syntax was introduced: https://github.com/ocaml/opam-repository/pull/25121/files#diff-50dbda423c7e82cdea4d115977ce0d08ff4c4c0cbb30bb80db4b9dda21f957d1R26

The new error relates to the main library:
```suggestion
  "ocaml" {>= "4.03"}
```